pkg/kubeclient/client_test.go

```diff
@@ -0,0 +1,106 @@
+package kubeclient_test
+
+import (
+	"context"
+
+	"github.com/kubernetes-sigs/kubebuilder/pkg/kubeclient"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ExampleClient_Create() {
```

Comment:
Feel free to document your example by adding a comment befor e.

Reply:
Collapsed all the examples in one example because there was lot of repetition. Also added comments.